### ISSUE TITLE
Hiding Auto Generated YouTube Captions

### ISSUE THREAD

**Issue by richardfrost** [richardfrost/AdvancedProfanityFilter#190]
#### :loudspeaker: The Shortcoming
Hide YouTube captions, even on auto-generated videos.

#### :microscope: Describe the solution you'd like
It would be nice to give users the option to hide captions on YouTube videos that have auto-generated subtitles.

#### :pencil: Additional context
This was requested in a Chrome Web Store support request.

**Comment by richardfrost** [comment]
This will require some investigation.

**Comment by richardfrost** [comment]
Alright, it should be possible, but as of right now, it might be an all-or-nothing deal: hide all subtitles, or none.

Its far too jarring to have the whole row(s) of text appear suddenly, only to be hidden again quickly. And even with just showing the word being filtered, it still seems to hard to follow to me. By the time your eyes track down where you should be reading, its gone. Unless I hear from anyone else with another idea, or another way to look at this, I'll probably just move forward with adding the ability to hide captions for YouTube auto-generated captions.

**Comment by JayH2971** [comment]
I'm completely happy with the ability to hide all captions. I don't need to read them; it's the audio filtering that's so amazing. Thank you!

**Comment by richardfrost** [comment]
Cool! Thank you for your input @JayH2971. I'm glad that you are finding the filter useful.

What I ended up doing is setting YouTube auto-sub videos to hide all subtitles unless the user has set the option to "Show all" subtitles in the audio tab. To me that seems like the desired behavior, but I'm open to other ideas to improve it in the future.

**Comment by richardfrost** [comment]
Released with [2.6.1](https://github.com/richardfrost/AdvancedProfanityFilter/releases/tag/2.6.1).

If you notice any trouble after getting the update, feel free to comment and re-open this issue.
